addons/website_slides/tests/test_attendee.py

```diff
@@ -441,10 +441,15 @@ def test_invite_route_errors_handling(self):
         self.channel.visibility = 'members'
 
         # No such channel
```

Comment:
Rename to `# slugification can give: "/slides/-ID" which should work, despite resulting in a negative ID`